src/output/localization.js

```diff
@@ -77,4 +77,19 @@ const getLocalization = () => {
   }
   return localization.en;
 };
-module.exports = { getLocalization };
+
+const translate = stringToTranslate => {
```

Comment:
Yep I can see what you're going for here and it does make it a bit safer - return what was passed in if nothing was found.
Could potentially be shortened a bit like:
`return getLocalization()[stringToTranslate] || stringToTranslate`
We could also throw an error instead because this is a programmer error - we haven't defined the stringToTranslate in our localization file, and then the tests would catch this before commit